src/components/QandA/QuestionList.jsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 
-const QuestionList = (props) => {
+function QuestionList ({count, questions}) {
 
+  //console.log('questions = ', questions);
+  //console.log('questions.results = ', questions.results);   // default: 4
 
-  // console.log('props = ', props)
-  // console.log('props = ', props);
-  // console.log('props.questions = ', props.questions);
-  // console.log('props.count = ', props.count);   // default: 4
-
-  const last = props.count;
+  const last = count;
 
   return (
+    <div>
```

Comment:
Be sure to wrap this all in a single dom element. Recommend using <React.Fragment>